hdl/ip/vhd/common/BUCK

```diff
@@ -1,38 +1,23 @@
 load("//tools:hdl.bzl", "vhdl_unit")
 
-# Utilities
-
-vhdl_unit(
-    name = "calc_pkg",
-    srcs = ["utils/calc_pkg.vhd",],
-    visibility = ['PUBLIC'],
-)
+# Countdown
 
 vhdl_unit(
-    name = "time_pkg",
-    srcs = ["utils/time_pkg.vhd",],
-    visibility = ['PUBLIC'],
-)
-
-vhdl_unit(
-    name = "transforms_pkg",
-    srcs = ["utils/transforms_pkg.vhd",],
+    name = "countdown",
+    srcs = glob(["countdown/countdown.vhd"]),
+    deps = [],
     visibility = ['PUBLIC'],
 )
 
 vhdl_unit(
-    name = "utilities_tb",
+    name = "countdown_tb",
     is_tb = True,
-    srcs = glob(["utils/sims/*.vhd"]),
-    deps = [
-        ":transforms_pkg", 
-        ":time_pkg", 
-        ":calc_pkg",
-    ],
+    srcs = glob(["countdown/sims/*.vhd"]),
+    deps = [":countdown"],
     visibility = ['PUBLIC'],
 )
 
-# Interfaces
+# Interfaces - Note these packages require VHDL-2019 support
```

Comment:
Does this add value vs having it defined below?

Reply:
Probably not!